src/index.js

```diff
@@ -707,6 +721,16 @@ class State extends EventEmitter {
 	}
 
 	/**
+	 * Send an arbitrary message to the named channel. The type of the message will be coerced to
+	 * a Max type in the Max application by mira.channel
+	 * @param {String} channel - The name of the mira.channel objects that should receive this message
+	 * @param {Number|String|Array<Number|String>|Object} message - the message to send
+	 */
+	sendMessageToChannel(channel, message) {
```

Comment:
Well, it's not a network error or an asynchronous error---it's only a problem in trying to change the message into a Max message that will throw the error. That's why I went with a throw rather than a callback. Is there some other reason that a callback makes more sense?